packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/tests/rich-text-mention-users-view.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { html } from '@microsoft/fast-element';
+import { RichTextMentionUsersView, richTextMentionUsersViewTag } from '..';
+import { type Fixture, fixture } from '../../../../../utilities/tests/fixture';
+import { waitForUpdatesAsync } from '../../../../../testing/async-helpers';
+
+async function setup(): Promise<Fixture<RichTextMentionUsersView>> {
+    return fixture<RichTextMentionUsersView>(
+        html`<nimble-rich-text-mention-users-view
+            mention-href="users:1"
+            mention-label="John Doe"
+            disable-editing
+            >@John Doe</nimble-rich-text-mention-users-view
+        >`
+    );
+}
+
+describe('RichTextMentionUsersView', () => {
+    let element: RichTextMentionUsersView;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+
+    beforeEach(async () => {
+        ({ element, connect, disconnect } = await setup());
+    });
+
+    afterEach(async () => {
+        await disconnect();
+    });
+
+    it('should export its tag', () => {
+        expect(richTextMentionUsersViewTag).toBe(
+            'nimble-rich-text-mention-users-view'
+        );
+    });
+
+    it('can construct an element instance', () => {
+        expect(
+            document.createElement('nimble-rich-text-mention-users-view')
+        ).toBeInstanceOf(RichTextMentionUsersView);
+    });
+
+    it('should set the "control" class on the internal control', async () => {
+        await connect();
+        expect(
+            element.shadowRoot?.firstElementChild?.classList.contains('control')
+        ).toBeTrue();
+    });
```

Comment:
This test isn't useful. Those classes are private to the shadow root. Nothing external should be aware of them

Reply:
I have removed this test